Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "tinytga"
+version = "0.3.2"
+description = "No-std, low memory footprint TGA image loader"
+authors = ["James Waples <james@wapl.es>"]
+edition = "2018"
+repository = "https://github.com/embedded-graphics/embedded-graphics/tree/master/tinytga"
+documentation = "https://docs.rs/tinytga"
+categories = ["embedded", "no-std"]
+keywords = ["graphics", "embedded", "tga", "targa", "image"]
```

Comment:
Should we start adding a `embedded-graphics` keyword to all crates in the e-g ecosystem? This way we could add a link to https://crates.io/keywords/embedded-graphics in the README to list all e-g related crates. e-t and e-l already include this keyword.

Reply:
That's a very good idea - I'll add it to all the new repos